docs/learn/learn-account-multisig.md

```diff
@@ -158,13 +166,38 @@ video tutorials for more information about creating multisig accounts and transa
 using both the [Accounts Tab](https://www.youtube.com/watch?v=-cPiKMslZqI) and the
 [Extrinsic Tab](https://www.youtube.com/watch?v=T0vIuJcTJeQ) in the Polkadot-JS UI.
 
+## Multisig with Multix Tool
+
+:::info Walk-Through Video Tutorial
+
+See [this video tutorial](https://www.youtube.com/watch?v=APxPsawebJw) and
+[this article](https://blog.chainsafe.io/multix-a-simple-interface-to-manage-complex-multisigs-on-polkadot-97328be26f9d)
+for more information about using [Multix](https://multix.chainsafe.io/) for multisig accounts.
+
+:::
+
+> Note that Multix is currently only available on Rococo (Polkadot's Testnet). Soon it will be
```

Comment:
Thanks for the shoutout. You can remove this, it will be outdated in the next days anyway.